druid/src/core.rs

```diff
@@ -468,7 +495,9 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
             let mut visible = ctx.region().clone();
             visible.intersect_with(self.state.paint_rect());
             visible -= layout_origin;
+            ctx.native_origin += layout_origin;
             ctx.with_child_ctx(visible, |ctx| self.paint_raw(ctx, data, env));
+            ctx.native_origin -= layout_origin;
```

Comment:
Wouldn't it make more sense to modify this inside the closure to with_child_ctx so we don't need to back it out?

Reply:
Do you mean add `native_origin` as a parameter to `with_child_ctx()`, and perform the modification after saving the context?